python/llm/example/NPU/HF-Transformers-AutoModels/LLM/README.md

```diff
@@ -77,45 +76,9 @@ Inference time: xxxx s
 done
 ```
 
-## Example 2: Predict Tokens using `generate()` API using multi processes
+## 4. Run Optimized Models (Experimental)
 In the example [llama2.py](./llama2.py) and [qwen2.py](./qwen2.py), we show an experimental support for a Llama2 / Qwen2 model to predict the next N tokens using `generate()` API, with IPEX-LLM INT4 optimization and fused decoderlayer optimization on Intel NPUs.
```

Comment:
The examples below shows how to run the optimized model implementations on Intel NPU, including
- [Llama2-7B](./llama2.py)
- [Qwen2-1.5B](./qwen2.py)

Reply:
> The example below shows how to run the **_optimized model implementations_** on Intel NPU, including
> 
> * [Llama2-7B](./llama2.py)
> * [Qwen2-1.5B](./qwen2.py)

Have updated.